### ISSUE TITLE
Cannot load Valine comments data with PJAX

### ISSUE THREAD

**Issue by lei2rock** [theme-next/hexo-theme-next#1055]
### I agree and want to create new issue 

- [X] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [X] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [X] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
**After adding PJAX support:** load Valine comments data when visit posts and pages for the first time from home page

### Actual behavior 
see the next part (Steps to reproduce the behavior)

### Steps to reproduce the behavior 
_**After adding PJAX support:**_
1. visit https://blog.dlzhang.com
2. click menu item "关于更多", **cannot** load Valine comments data automatically
3. refresh the page ("关于更多"), load Valine comments data successfully
4. If we do not refresh the page (without Step 3), after Step 2, click the above sub-menu item "本站概览", also load Valine comments data successfully.
5. If we visit https://blog.dlzhang.com/about/ directly, load Valine comments data successfully.
6. If we click the number of comments under post title on homepage, **cannot** load Valine comments data automatically

* Link to demo site with this bug: https://blog.dlzhang.com
* Link(s) to source code or any usefull link(s): https://github.com/lei2rock/blog

 ### Node.js and NPM Information

```
v12.1.0
6.10.0
```

### Package dependencies Information

```
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "hexo": {
    "version": "3.9.0"
  },
  "dependencies": {
    "hexo": "^3.9.0",
    "hexo-abbrlink": "^2.0.5",
    "hexo-deployer-git": "^1.0.0",
    "hexo-generator-archive": "^0.1.5",
    "hexo-generator-category": "^0.1.3",
    "hexo-generator-index-pin-top": "^0.2.2",
    "hexo-generator-searchdb": "^1.0.8",
    "hexo-generator-sitemap": "^1.2.0",
    "hexo-generator-tag": "^0.2.0",
    "hexo-renderer-ejs": "^0.3.1",
    "hexo-renderer-marked": "^1.0.1",
    "hexo-renderer-stylus": "^0.3.3",
    "hexo-server": "^0.3.3",
    "image-size": "^0.7.4"
  }
}

```

#### Hexo version

```
hexo: 3.9.0
hexo-cli: 2.0.0
os: Darwin 18.7.0 darwin x64
node: 12.1.0
v8: 7.4.288.21-node.16
uv: 1.28.0
zlib: 1.2.11
brotli: 1.0.7
ares: 1.15.0
modules: 72
nghttp2: 1.38.0
napi: 4
llhttp: 1.1.1
http_parser: 2.8.0
openssl: 1.1.1b
cldr: 35.1
icu: 64.2
tz: 2019a
unicode: 12.1
```

#### Hexo Configuration

https://github.com/lei2rock/blog/blob/src/_config.yml

### NexT Information

**NexT Version:**

- [X] Latest Master branch
- [ ] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [ ] All schemes
- [ ] Muse
- [ ] Mist
- [ ] Pisces
- [X] Gemini

#### NexT Configuration:

https://github.com/lei2rock/blog/blob/src/source/_data/next.yml

**Comment by stevenjoezhang** [comment]
Can't reproduce on my site: https://zhangshuqiao.org

You can try the newly installed theme (without custom files), and check if the issue still exists

**Comment by lei2rock** [comment]
@stevenjoezhang I tried a newly installed theme `hexo init new_blog`, and enable Valine & PJAX. The issue still exits:

1. visit the homepage
2. click a post with comments, but the comments do not display. 
3. Then, refresh the page or click another post with comments, the comments can display successcully.

ps: my LeanCloud app is international version.

**Comment by stevenjoezhang** [comment]
I found a more robust way to load comments... that should fix the problem

**Comment by sli1989** [comment]
i thins this commit make valine is unavailable, leading to site's lib js.

```
GET http://localhost:3000/lib/unpkg.com/valine/dist/Valine.min.js net::ERR_ABORTED 404 (Not Found)
```

**Comment by stevenjoezhang** [comment]
@sli1989 Fixed in https://github.com/theme-next/hexo-theme-next/commit/1bfd387d9c034ddbf9d40c25e8f4fe2ed3258805
